README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Mark Gacoka's portfolio and SPS projects
 
 To get started:
```

Comment:
Can probably remove this "To get started" section since it doesn't apply to people viewing your portfolio repo (it's an instruction for yourself to set up this portfolio)